Library/Homebrew/formula.rb

```diff
@@ -2918,8 +2918,34 @@ def service(&block)
     #
     # If `satisfy` returns `false` then a bottle will not be used and instead
     # the {Formula} will be built from source and `reason` will be printed.
-    def pour_bottle?(&block)
+    #
+    # Alternatively, a preset reason can be passed as a symbol:
+    # <pre>pour_bottle? :default_prefix_required</pre>
+    # <pre>pour_bottle? :clt_required</pre>
+    def pour_bottle?(requirement = nil, &block)
       @pour_bottle_check = PourBottleCheck.new(self)
+
+      block ||= case requirement
+      when :default_prefix_required
+        lambda do |_|
```

Comment:
```suggestion
        lambda do
```

Does this work?

Reply:
No, I get this error:

```
FormulaUnreadableError (hello: wrong number of arguments (given 1, expected 0))
```

According to the [`instance_eval` docs](https://ruby-doc.org/core-2.5.1/BasicObject.html#method-i-instance_eval):

> When `instance_eval` is given a block, _obj_ is also passed in as the block's only argument.